test/e2e/flask/multichain-api/create-session.spec.ts

```diff
@@ -261,4 +261,144 @@ describe('Multichain API', function () {
       });
     });
   });
+
+  describe('Connect wallet to the multichain dapp via `externally_connectable`, call `wallet_createSession` without any accounts requested', function () {
+    it('should automatically select first account', async function () {
+      await withFixtures(
+        {
+          title: this.test?.fullTitle(),
+          fixtures: new FixtureBuilder().withPopularNetworks().build(),
+          ...DEFAULT_MULTICHAIN_TEST_DAPP_FIXTURE_OPTIONS,
+        },
+        async ({
+          driver,
+          extensionId,
+        }: {
+          driver: Driver;
+          extensionId: string;
+        }) => {
+          await openMultichainDappAndConnectWalletWithExternallyConnectable(
+            driver,
+            extensionId,
+          );
+
+          await initCreateSessionScopes(driver, ['eip155:1']);
+
+          const editButtons = await driver.findElements('[data-testid="edit"]');
+          await editButtons[0].click();
+
+          const checkboxes = await driver.findElements(
+            'input[type="checkbox" i]',
+          );
+          const accountCheckbox = checkboxes[1];
+          const isChecked = await accountCheckbox.isSelected();
+
+          assert.strictEqual(
+            isChecked,
+            true,
+            'first account should be automatically selected',
```

Comment:
it's technically the currently active/selected account in the wallet is the one that is default, not necessarily the first one. I don't think we need to specifically check what the selected account is though.

Let's change this assert message though

Reply:
```suggestion
            'current active account in the wallet should be automatically selected',
```

Reply:
resolved in [6e0626b](https://github.com/MetaMask/metamask-extension/pull/29400/commits/6e0626b85a265db8e150209a2ae91b1a1f18ee04)